packages/audio-block/src/AudioBlock.spec.ct.tsx

```diff
@@ -0,0 +1,151 @@
+        cy.get(UploadPlaceholderSelector).should('not.exist');
+    });
+
+    it('renders an empty audio block in edit mode', () => {
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock, {
+            editorState: true,
+        });
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioBlockSelector).should('exist');
+        cy.get(UploadPlaceholderSelector).should('exist');
+    });
+
+    it('renders an audio block with an audio asset in view mode', () => {
+        const asset = AssetDummy.with(312);
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock, {
+            blockAssets: {
+                [AUDIO_ID]: [asset],
+            },
+        });
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioTagSelector).should('exist');
+    });
+
+    it('renders an audio block with an audio asset in editor mode', () => {
+        const asset = AssetDummy.with(312);
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock, {
+            editorState: true,
+            blockAssets: {
+                [AUDIO_ID]: [asset],
+            },
+        });
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioTagSelector).should('exist');
+    });
+
+    it('renders an audio block with an audio asset in view mode with title', () => {
+        const asset = AssetDummy.with(312);
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock, {
+            blockSettings: {
+                title: 'Audio Title',
+            },
+            blockAssets: {
+                [AUDIO_ID]: [asset],
+            },
+        });
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioBlockSelector).contains('Audio Title').should('exist');
+    });
+
+    it('renders an audio block with an audio asset in view mode with description', () => {
+        const asset = AssetDummy.with(312);
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock, {
+            blockSettings: {
+                description: 'Audio Description',
+            },
+            blockAssets: {
+                [AUDIO_ID]: [asset],
+            },
+        });
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioBlockSelector).contains('Audio Description').should('exist');
+    });
+
+    it('renders an audio block with an audio asset in view mode with title and description', () => {
+        const asset = AssetDummy.with(312);
+        const [AudioBlockWithStubs] = withAppBridgeBlockStubs(AudioBlock, {
+            blockSettings: {
+                title: 'Audio Title',
+                description: 'Audio Description',
+            },
+            blockAssets: {
+                [AUDIO_ID]: [asset],
+            },
+        });
+
+        mount(<AudioBlockWithStubs />);
+        cy.get(AudioBlockSelector).contains('Audio Title').should('exist');
+        cy.get(AudioBlockSelector).contains('Audio Description').should('exist');
```

Comment:
Don't rely on strings for tests, please add a `data-test-id` on both title and description
Same for all other hardcoded strings

Reply:
adjusted within [commit](https://github.com/Frontify/guideline-blocks/pull/475/commits/d5db1bb55113c9f03cf16bd62101414ab00bc979)